### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by potatomioo** [community/community#33628] [GitHub Education]
I have registered for GitHub Campus Expert for my campus. And I got an information from GitHub that I am going to get a mail related to it till 19th September 2022. But I did not get any update yet. So I just wanted to get an update related to it.

**Comment by samus-aran** [comment]
Hi @x-deeptanshu, another community member has helped answering this in a [similar discussion](https://github.com/orgs/community/discussions/33668#discussioncomment-3698173). 

> The results of applications submitted in the month of Aug-Sep 2022 are yet to be released. Keep check on your mailbox and the social handle of [Github Education on twitter ](https://twitter.com/GitHubEducation)for updates.
